Libraries/PeriphDrivers/Source/AFE/afe.c

```diff
@@ -536,6 +552,86 @@ static int afe_spi_transceive(uint8_t *data, int byte_length)
     // Got all bytes, and we did NOT timeout
     return E_NO_ERROR;
 }
+#else // Not AFE_SPI_TRANSCEIVE_SAFE_BUT_SLOWER
+
+//
+// This function blocks until transceive is completed, or times out
+//  This version does not use the afe timer to interrupt potential infinite polling.
+//
```

Comment:
Does this version actually time out?

Reply:
It does not time out.  Thanks, will update comment.